### PR TITLE
Add Jordi Creus to employee list

### PR DESCRIPTION
**Pull request by ghost**
- add a new team biography for Jordi
- add bio image for Jordi

### CODE REVIEW

File: src/_data/team.yml [hunk 1]
```diff
@@ -624,3 +624,13 @@
...
   github: RoryCollins
   twitter: Rory_Collins_
+
+- id: jordi_creus
+  name: Jordi Creus
+  image: /assets/custom/img/team/jordi_creus.jpg
+  role: Software Craftsman
+  fullDescription:
+    <p>Jordi is a software developer specialised in the web environment. He practices agile methodologies and he is always willing to learn new technologies and methodologies.</p>
+    <p>He have extensive experience creating and defining the architecture of web applications from scratch, as well as controlling the technical debt and legacy code.</p>
```
Comment (flyinggab) [inline src/_data/team.yml]: I think the correct form is "_He has extensive_"

**Review by flyinggab — CHANGES_REQUESTED** [review]

File: src/_data/team.yml (hunk 1 above)
Comment (ghost) [reply src/_data/team.yml]: Thanks, should be fixed now

**Comment by Mattsi-Jansky** [comment]
Few more minor English grammar things: "specialised in the web environment" -> "specialised in web development"

"He practices agile methodologies and he is always" -> "He practices agile methodologies and is always"

"controlling the technical debt" -> "controlling technical debt"

**Review by flyinggab — APPROVED** [review]

**Comment by flyinggab** [comment]
Thanks @Mattsi-Jansky I was not sure of those

**Pull request merged**